lib/util/stylesheet.js

```diff
@@ -330,10 +357,13 @@ const astHelpers = {
   },
 
   hasArrayOfStyleReferences: function (node) {
-    return node && Boolean(
-      node.type === 'JSXExpressionContainer' &&
-      node.expression &&
-      node.expression.type === 'ArrayExpression'
+    return (
+      node &&
+      Boolean(
```

Comment:
I guess this is just a Prettier/ESLint formatting fix, but how about moving the `Boolean` call up a level, like so:

```
return Boolean(
  node &&
  node.type === 'JSXExpressionContainer' &&
  node.expression &&
  node.expression.type === 'ArrayExpression'
)
```

Reply:
i've reverted the formatting change, we probably should do this in another PR?